<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageWidget</class>
 <widget class="QWidget" name="MessageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{border-image: url(:/images/border.png);}</string>
     </property>
     <widget class="QPushButton" name="okBtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
	background-color: rgb(100, 180, 255);
}
QPushButton:pressed {
background-color: #1E90FF;
}
QPushButton {
	background-color: rgb(64, 169, 255);
	color: white;                    /* 文字颜色 */
	border-width: 1px;               /* 边框宽度 */
	border-radius: 2px;              /* 边框圆角 */
	font:bold 10pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton:!enabled {
background-color: rgb(150, 150, 150);
}
</string>
      </property>
      <property name="text">
       <string>接受</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="noBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover {
	background-color: rgb(200, 200, 200);
}
QPushButton:pressed {
background-color:  rgb(160, 160,160);
}
QPushButton {
	background-color: rgb(180, 180, 180);
	color: white;                    /* 文字颜色 */
	border-width: 1px;               /* 边框宽度 */
	border-radius: 2px;              /* 边框圆角 */
	font:bold 10pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton:!enabled {
background-color: rgb(150, 150, 150);
}
</string>
      </property>
      <property name="text">
       <string>拒绝</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
